src/label_studio_sdk/label_interface/interface.py

```diff
@@ -300,7 +300,7 @@ def __init__(self, config: str, tags_mapping=None, *args, **kwargs):
         self._labels = labels
         self._tree = tree
 
-    def create_regions(self, data: Dict[str, Union[Dict, List[Dict]]]) -> List[Region]:
+    def create_regions(self, data: Dict[str, Union[Dict, List[str], List[Dict]]]) -> List[Region]:
```

Comment:
I'm guessing this is just a miss in the type annotation, but seems we should also accept a `str` as a value type here, yeah? For single label?

Reply:
Seems like it yeah adding now